docs/install-upgrade.md

```diff
@@ -4,6 +4,7 @@
 
 - [Install the Operator](#install-the-operator)
   - [Prerequisites](#prerequisites)
+  - [Understand MongoDB Community Kubernetes Operator Deployment Scopes](#understand-mongodb-community-operator-deployment-scopes)
```

Comment:
When I click on this link I don't get directed to the new section. Is something off here?